README.md

```diff
@@ -1,3 +1,9 @@
 # TODO
 
 * Write readme
+
+# Add a group
+* Add group in IM (either through the UI or by using the user script scripts/users/createGroup.sh)
+* Update [values file](https://github.com/dhis2-sre/im-manager/blob/8cb9a5959e334b835188fa07e801996ff2410b7c/helm/chart/values.yaml#L12) or for an individual environment such as [prod](https://github.com/dhis2-sre/im-manager/blob/8cb9a5959e334b835188fa07e801996ff2410b7c/helm/data/values/prod/values.yaml#L1).
```

Comment:
Should use a URL that points to `master`, not a specific commit (applies to all URLs below, too). 

And should we even suggest making changes to the main values file of the chart?

Reply:
> Should use a URL that points to `master`, not a specific commit (applies to all URLs below, too).

With a permanent link there's no doubt about what we're referring to. If things move around it should be easy to find in the lastest version since the user knows exactly what they're looking for.

> And should we even suggest making changes to the main values file of the chart?

Depends on the use case I suppose. I'm just making aware of it.